src/Modules/CalcSetup.lua

```diff
@@ -738,6 +738,10 @@ function calcs.initEnv(build, mode, override, specEnv)
 						-- checks if it disables another slot
 						for _, tag in ipairs(mod) do
 							if tag.type == "DisablesItem" then
+								-- e.g. Tincture in Flask 5 while using a Micro-Distillery Belt
+								if tag.exclusion and items[tag.slotName] and items[tag.slotName].name:find(tag.exclusion) then
```

Comment:
Maybe base this on item.type instead of item.name?

![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/1209372/688d9351-fd68-4fa9-ad15-1f3e85d603a6)

Reply:
![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/92683202/96c5d68b-252b-4cb1-a4d6-0d3afe48e6f1)